loader/src/packfile_io.rs

```diff
@@ -140,11 +141,13 @@ impl PackfileReader {
                 .or_insert_with(|| vec![idx as u32]);
         }
 
+        let runtime = tokio::runtime::Builder::new_current_thread().build()?;
```

Comment:
Is it possible to pass a `use single threaded` flag somehow? e.g. when configuring loaders or `distill`